apps/relayer/package.json

```diff
@@ -46,9 +46,9 @@
     "helmet": "^8.0.0",
     "lodash": "^4.17.21",
     "maci-cli": "workspace:^2.5.0",
-    "maci-contracts": "workspace:^2.5.0",
     "maci-sdk": "workspace:^0.0.1",
     "maci-domainobjs": "workspace:^2.5.0",
+    "maci-contracts": "workspace:^2.5.0",
```

Comment:
Do we still need it?

Reply:
looks like it's needed for the contract artifacts on hardhat config, not sure whether we can reference from sdk there, or add as dev dep only?

Reply:
Right, let's keep it in deps because we need artifacts to call message publishing.